docs/index.rst

```diff
@@ -11,6 +11,30 @@ Everything else is on
 `Github Wiki <https://github.com/Ulauncher/Ulauncher/wiki/>`_.
 
 
+Launch-Targets
+==============
+
+Additionally to launching Applications, Ulauncher can launch the following targets:
+
+Desktop-Entries
+---------------
+
+Ulauncher indexes desktop-entries in ``~/.local/share/applications`` the same as applications.
+Using desktop-entries, launchers that require no query (no space or arguments other than the name/keyword) can be created easily.
+
+
+Shortcuts
```

Comment:
@kstenschke ... `Shortcuts` terminalogy applies to lauching both web searches and extensions so this would be confusing as well.